Makefile

```diff
@@ -1308,8 +1308,8 @@ ui-lint: pkg/ui/yarn.installed $(UI_PROTOS_OSS) $(UI_PROTOS_CCL)
 
 # DLLs are Webpack bundles, not Windows shared libraries. See "DLLs for speedy
 # builds" in the UI README for details.
-UI_CCL_DLLS := pkg/ui/dist/protos.ccl.dll.js pkg/ui/dist/vendor.oss.dll.js
-UI_CCL_MANIFESTS := pkg/ui/protos.ccl.manifest.json pkg/ui/vendor.oss.manifest.json
+UI_CCL_DLLS := pkg/ui/dist/vendor.oss.dll.js
+UI_CCL_MANIFESTS := pkg/ui/vendor.oss.manifest.json
```

Comment:
Is it a naming mismatch for variable and library? UI_**CCL**_MANIFESTS --> pkg/ui/vendor.**oss**.manifest.json
CCL in variable name refers to OSS file name.

Reply:
Thanks @Koorosh nice catch, that does look like an naming mismatch!

Reply:
After taking a closer look, this is, in fact, correct. The vendor file is *always* marked as `oss` since everything in it is open source. I've updated the Makefile to rename these variables and make that clearer.